### DISCUSSION TITLE
M365 Group Calendar in SharePoint and/or MS Teams

### DISCUSSION THREAD

**Discussion by ismaili-de** [pnp/sp-dev-fx-webparts#2154] [Q&A]
I am looking for a possibilitiy to show a calendar from a m365 group in SPO or MS Teams.
I thought, such a requirement should be pretty common, but it seems, that this is not the case and not supported by any capability that I know...
The ootb SPO group calendar webpart shows only next few events, not a calendar view.
Microsoft seems to have removed the possibility to show a group calendar with a sepcific URL, so I cannot have a Teams tab with the group calendar (?)
I was hoping that this calendar solution would solve my problem, but this does not seem to be the case, as it seems, it accesses only SharePoint calendar lists ...
Any ideas?

**Comment by amirmsai** [comment]
Hi 
What is with graph; if you call all events !!
Amir MSAI

**Reply by ismaili-de** [reply]
Thank you, Amir, because of your response, I looked again at the available samples, and indeed, there seems to be the "react-graph-calendar", which could probably fulfill my needs.

A first try to get it to run, failed (I got the calendar to show up in MS Teams, but no events were listed in there), ... but I will check it out further and investigate a little. It seems, nobody touched it for some time ...
